pkg/local_object_storage/engine/delete.go

```diff
@@ -27,6 +30,8 @@ func (p *DeletePrm) WithAddresses(addr ...*addressSDK.Address) *DeletePrm {
 // Delete marks the objects to be removed.
 //
 // Returns an error if executions are blocked (see BlockExecution).
+//
+// Returns apistatus.ObjectLocked if at least one object is locked.
```

Comment:
But deletes all objects that are not locked?

Reply:
What do u mean? Ur statement follows from method documentation.

Reply:
it is not clear from documentation, i think

i got such description as "if at least one object is locked, I will not delete any of them" or "... stop object iteration", but when I looked at method I saw that it would try to delete all the objects anyway (correct me if I'm wrong)

Reply:
btw, is it ok to return error that does not have any information how many object were deleted, how many are locked?

Reply:
Got it, I'll mention that in this case no object is deleted.